src/main/java/eu/europeana/metis/sandbox/config/amqp/AmqpConfiguration.java

```diff
@@ -125,97 +125,28 @@ Declarables deadQueues() {
 
   @Bean
   Declarables bindings() {
-    var queues = new Queues()
-        .setCreated(createdQueue)
-        .setExternalValidated(externalValidatedQueue)
-        .setTransformed(transformedQueue)
-        .setInternalValidated(internalValidatedQueue)
-        .setNormalized(normalizedQueue)
-        .setEnriched(enrichedQueue)
-        .setMediaProcessed(mediaProcessedQueue)
-        .setIndexed(indexedQueue);
-    return getDeclarables(queues);
+    return getDeclarables(createdQueue, externalValidatedQueue, transformedQueue,
+        normalizedQueue, internalValidatedQueue, enrichedQueue, mediaProcessedQueue, indexedQueue);
   }
 
   @Bean
   Declarables dlqBindings() {
-    var dlqs = new Queues()
-        .setCreated(createdDlq)
-        .setExternalValidated(externalValidatedDlq)
-        .setTransformed(transformedDlq)
-        .setInternalValidated(internalValidatedDlq)
-        .setNormalized(normalizedDlq)
-        .setEnriched(enrichedDlq)
-        .setMediaProcessed(mediaProcessedDlq)
-        .setIndexed(indexedDlq);
-    return getDeclarables(dlqs);
+    return getDeclarables(createdDlq, externalValidatedDlq, transformedDlq,
+        normalizedDlq, internalValidatedDlq, enrichedDlq, mediaProcessedDlq, indexedDlq);
   }
 
-  private Declarables getDeclarables(Queues qs) {
+  private Declarables getDeclarables(String created, //NOSONAR
```

Comment:
Sonar complains about too many parameters, but if I split this into 2 methods and use the instance variables, the it will complain about duplicate code. So from those I pick this one and ignore Sonar

Reply:
Ok, but I would say to add a specific sonar issue ID here, so that if another issue is triggered, we don't miss it. You can do this like so (replace the string argument by the ID of your specific sonarQube issue):
```
@SuppressWarnings("findbugs:RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE")
```
And perhaps put some comments in the code to explain the reason.

Reply:
Nice, I will change it